_includes/reference-code.md

```diff
@@ -0,0 +1,74 @@
+## Reference Code
+
+Swedbank Pay's Checkout v3 API is built in a way that depends on that you, as a
+merchant, is able to store the `paymentOrderId`. The `paymentOrderId` is a url
+which points to the specific instance of a payment resource, but in some cases
+it will be hard to use it as a payment reference.
+
+To help you keep track of the transaction easier, we provide a unique
+identification called `referenceCode` for all v3 implementations. The code can
+then be printed and used, for instance, as a scannable barcode for future
+tracking.
+
+The `referenceCode` isn't available by default, but needs to be activated for
+your merchant in our end to appear in the payment order response.
+
+## GET The Reference Code
+
+When enabled, the `referenceCode` will appear as a 16 digit code in the expanded
+`paid` node when the payment has been completed by the payer.
+
+There are two possibilities to choose from when you want to fetch it. The `GET`
+endpoint will be the same as with a regular `GET` call against the
+`paymentOrder` resource, but you can either use `referenceCode` or `paid` as an
+identification. Both responses will look like the abbreviated `paid` example
+below. The parts omitted from this example will look like a regular `GET`
+response.
```

Comment:
We don't have endpoints, we have resources. This needs to be rewritten and also clarified. How can you use `referenceCode` to retrieve a payment (order)? I have rewritten the section to be more correct, but it is still missing information on how to "use `referenceCode` or `paid` as an identification". 

```suggestion
There are two possibilities to choose from when you want to fetch it. The request
will look the same a regular `GET` request against the `paymentOrder` resource,
but you can either use `referenceCode` or `paid` as an identification. Both 
responses will look like the abbreviated `paid` example below. The parts omitted
from this example will look like a the response from a regular `GET` request.
```